docs/pipelineruns.md

```diff
@@ -38,11 +38,16 @@ following fields:
 
   - [`resources`](#resources) - Specifies which
     [`PipelineResources`](resources.md) to use for this `PipelineRun`.
-  - [`serviceAccount`](#service-account) - Specifies a `ServiceAccount` resource
+  - [`serviceAccountName`](#service-account) - Specifies a `ServiceAccount` resource
     object that enables your build to run with the defined authentication
+<<<<<<< HEAD
     information. When a `ServiceAccount` isn't specified, the `default-service-account`
     specified in the configmap - config-defaults will be applied.
   - [`serviceAccounts`](#service-accounts) - Specifies a list of `ServiceAccount`
+=======
+    information.
+  - [`serviceAccountNames`](#service-accounts) - Specifies a list of `ServiceAccountName`
+>>>>>>>  ServiceAccountNames replaces ServiceAccounts
```

Comment:
looks like a git merge artifact might have snuck in.

Reply:
fixed